README.md

```diff
@@ -1,105 +1,28 @@
-# Gobra docker action
+# Gobra Action
 
-This action verifies gobra files in a project
+This action adds Gobra to the CI Workflow of Go projects.
 
-## Inputs
+## Usage
 
-## `caching`
+See [action.yml](./action.yml)
 
-**Required** Wheather or not caching should be enabled (1) or disabled (0)
+## Examples
 
-*Default:*  0
-
-## `projectLocation`
-
-**Not Required** Location in which the project is located in the workflow context. Per default the action will look in a folder named the same as the repository(This matches the pull action's default behaviour).
-
-## `srcDirectory`
-
-**Required** Path to the project source directory relative to the `projectLocation`.
-*Default:* The projectLocation
-
-## `packageDirectories`
-
-**Not Required** lookup paths of go packages, relative to the `projectLocation`.
-*Default:* Same folder as `projectLocation`
-
-## `packages`
-
-**Not Required** Names of the packages that should be verified. If set ignores all packages not contained in this argument.
-
-## `excludePackages`
-
-**Not Required** Names of the packages that should NOT be verified. Takes precedence over the `packages` argument.
-
-## `chop`
-**Not Required** In how many pieces a viper program should be chopped for verification. May help verify very large packages
-
-*Default:* 1
-
-## `viperBackend`
-
-**Required** Which viper backend to use, one of `SILICON` or `CARBON`
-
-*Default:* `SILICON`
-
-## `javaXss`
-
-**Required** Java stack size, increase in case of stack overflows
-
-*Default:* 128m
-
-## `javaXmx`
-
-**Required** Java maximum heap size
-
-*Default:* 4g
-
-## `globalTimeout`
-
-**Required** Time till the action as a whole times out. Note that a GitHub workflow step times out automatically after 6 hours.
-
-*Default:* 6h
-
-## `packageTimeout`
-
-**Not Required** Time till the verification of a package times out.
-
-## `imageName`
-
-**Required** Which docker image should be used.
-
-*Default:* [ghcr.io/viperproject/gobra](https://github.com/viperproject/gobra/pkgs/container/gobra)
-
-## `imageVersion`
-
-**Required** Which image tag should be used.
-
-*Default:* latest
-
-## Outputs
-
-## `time`
-Total verification time in seconds.
-
-## Example usage
-
-### Without caching
+### Simple set-up without caching
 
 ```yaml
-uses: jogasser/gobra-action@main
+uses: viperproject/gobra-action@main
   with:
+  	caching: 0
     javaXss: 64m
     globalTimeout: 1h
     packageTimeout: 10m
-    packageLocation: gobra
+    packageLocation: go
```

Comment:
What is `packageLocation`? Is it explained in the options below?

Reply:
Good point. This was probably meant to be `projectLocation`